numpyro/distributions/transforms.py

```diff
@@ -159,6 +159,10 @@ def __call__(self, x):
         return jnp.abs(x)
 
     def _inverse(self, y):
+        warnings.warn(
+            "AbsTransform is not a bijective transform."
+            "The inverse of `y` will be `y`."
+        )
```

Comment:
Just a small suggestion (feel free to ignore of course :smile: ), so that the warning indicates where it's coming in the _user_'s code

```suggestion
        warnings.warn(
            "AbsTransform is not a bijective transform."
            "The inverse of `y` will be `y`.",
            stacklevel=3,
        )
```

Reply:
e.g. if I have a file `t.py` with
```python
import numpyro.distributions as dist

f1 = dist.transforms.AbsTransform()
f1.inv(-1)
```

then the current code shows:
```console
$ python t.py 
/home/marco/numpyro-dev/numpyro/distributions/transforms.py:162: UserWarning: AbsTransform is not a bijective transform.The inverse of `y` will be `y`.
  warnings.warn(
```
while with this change, it shows
```console
$ python t.py 
t.py:4: UserWarning: AbsTransform is not a bijective transform.The inverse of `y` will be `y`.
  f1.inv(-1)
```

Reply:
ah, this also highlights the missing whitespace before "The" :laughing:

Reply:
actually, it seems that all warnings in NumPyro are without `stacklevel` set, so perhaps that's best left to a separate PR

I'll just keep the suggestion here to add the extra whitespace then

Reply:
Thank you! I just merged and addressed the issue.